### ISSUE TITLE
MGS2S problems with rendering models

### ISSUE THREAD

**Issue by flcl8193** [AlpyneDreams/d8vk#213]
### Software information
Metal Gear Solid 2: Substance

Launched in winlator.

The game works fine except for glitches in all character models.

The game is launched with low model quality settings, otherwise the game has no problems with the model glitch, but everything else has no textures or the models are not displayed on the screen.

https://postimg.cc/vDNLxxfT

### System information
- GPU: Adreno 610
- Driver: Turnip Adreno (TM) 610
- Wine version: 8.0.2
- DXVK version: 1.0 (d8vk)

**Comment by WinterSnowfall** [comment]
I remember Alpyne already has fixed quite a few things for MGS2 and that it was working relatively correctly on Nvidia for the 1.0 release. See ##63 for more details.

That being said, does setting `d3d8.managedBufferPlacement = False` happen improve the situation in any way?

**Comment by flcl8193** [comment]
No. I deleted gog d3d8, created dxvk.conf in the folder with mgs2_sse.exe and set d3d8.managedBufferPlacement = False, but this did not change anything at all.
The game works with wined3d, but the fps is very low as software emulation

**Comment by WinterSnowfall** [comment]
> No. I deleted gog d3d8, created dxvk.conf in the folder with mgs2_sse.exe and set d3d8.managedBufferPlacement = False, but this did not change anything at all. The game works with wined3d, but the fps is very low as software emulation

Thanks for trying it out. Would appreciate if you could capture a trace of the offending section. I could then check if it's a problem on other drivers as well at least.

**Comment by flcl8193** [comment]
Maybe this is just my problem with my adreno 610. 
Here's the adreno 640 in the video and no problems https://m.youtube.com/watch?v=swCra9cwb88

**Comment by flcl8193** [comment]
Replacing the lines "GeForce" with "Removed", "RADEON" with "Turnip" in mgs2_sse.exe and setting high quality models completely solved the problem.

However, the introductory videos still do not work with the error "Could not add Decoder renderer filter to graph! hr=0x80004003".
